docs/backend-requests/making/same-origin.mdx

```diff
@@ -1,5 +1,97 @@
 ---
-sanity_slug: /docs/request-authentication/same-origin
+title: Same-origin requests
+description: Learn about various ways to make same-origin requests to the backend using the Clerk SDK.
```

Comment:
Okay it looks like we use `Clerk SDK` quite a bit. It sounds kind of awkward to me though...what do you think is better?

* Clerk SDK
* Clerk's SDK
* Clerk SDKs
* Clerk's SDKs

Reply:
We could cut SDK altogether and just say "Learn about various ways to make same-origin requests to the backend." or "Learn about various ways to make same-origin requests to the backend when using Clerk SDKs."

Reply:
I like "Clerk SDKs" 👍